### DISCUSSION TITLE
Window within w window on table row click

### DISCUSSION THREAD

**Discussion by hoffenmerCode** [beeware/toga#1272] [Q&A]
I'm trying to make a window within a MainWindow that opens on a button click. 
I'm pulling data from a db and listing in in a table and the idea is to open a new popup window with detailed view of a selected row. But I don't know how to have a second window template that would open from the MainWindow. 
I read all the BeeWare and Toga docs but am still lost on this one. 
Can someone please show a code sample that would achieve the above?

**Comment by freakboy3742** [comment]
Moving this to a discussion since it's a question, rather than a feature enhancement. 

If I'm understanding what you're asking for, the `dialogs` example contains an example of this - click on the "create secondary window" button, and a new window (with it's own layout) is created.
